File: Exercises/Ex5/returned_answers/Emma_van_Oijen/tree.cxx
void tree(){

  TFile * datafile = TFile::Open("random_data.root","RECREATE");
  TTree* mytree = new TTree("mytree","Gaussian random numbers");
  Double_t random_number;
  mytree->Branch("random_number",&random_number);

  int N = 1000;
  TRandom3 rand(0);

  for(int i = 0; i < N; ++i){
    random_number = rand.Gaus(0, 1);
    mytree->Fill();
  }

  mytree->Write();
  datafile->Close();
}

File: Exercises/Ex6/returned_answers/Emma_van_Oijen/tree.py
#!/usr/bin/env python

import ROOT
from array import array

def main():
  datafile = ROOT.TFile("random_data_py.root","RECREATE")
  mytree = ROOT.TTree("mytree","Gaussian random numbers")
  random_number = array('d', [0.0])
  mytree.Branch("random_number",random_number,"random_number/D")

  N = 1000;
  rand = ROOT.TRandom3(123456789);

  for i in range(0,N):
    random_number[0] = rand.Gaus(0, 1)
    mytree.Fill()

  mytree.Write()
  datafile.Close()

main()
